pylint_odoo/checkers/itp_checks.py

```diff
@@ -138,6 +143,29 @@ def _check_manifest_image(self):
             else:
                 return False
 
+    @utils.check_messages('readme-versions')
+    def _check_readme_versions(self):
+        readme_file = os.path.isfile(os.path.join(self.module_path, 'README.rst'))
+        if readme_file:
+            for line in self.readme_text:
+                urls = re.findall(
+                    'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+', line)
```

Comment:
Add a comment what does it look for all links or for some specific. If you copy-pasted it from stackoverflow, paste a linkt too

Reply:
https://www.tutorialspoint.com/python/python_extract_url_from_text.htm
It look all links

Reply:
Ok, but add a code-comment please